lib/nanoc/filters/colorize_syntax.rb

```diff
@@ -312,10 +312,36 @@ def coderay_postprocess(_language, element)
     def rouge(code, language, params = {})
       require 'rouge'
 
-      formatter_options = {
-        css_class: params.fetch(:css_class, 'highlight'),
-      }
-      formatter = Rouge::Formatters::HTML.new(formatter_options)
+      if Rouge.version < '2' || params.fetch(:legacy, false)
```

Comment:
we might change the logic here so that if we have Rouge 2.x and than neither `inline, linewise, pygments, table` is define and than there is arguments given then we fallback to legacy. That would prevent from explicitly specifying legacy.